### PR TITLE
Store pickup order placement from admin panel with Simple product automation.

### PR DESCRIPTION
**Pull request by nmalevanec**
### Description (*)

Test automation:
https://app.hiptest.com/projects/69435/test-plan/folders/971547/scenarios/3296742
[InStorePickupOrderPlacementWithSimpleProductAdditionalStockTest.xml](https://github.com/magento/inventory/blob/8a28981e7e8456d01bdde158fae321d84b96cbc7/app/code/Magento/InventoryAdminUi/Test/Mftf/Test/InStorePickupOrderPlacementWithSimpleProductAdditionalStockTest.xml) should be unskiped as soon as 
magento/inventory#2390: "Msi 2287 in store pickup orders placement from admin panel" will be finished and merged.
### Fixed Issues (if relevant)

1. magento/inventory#2399: MFTF - Store pickup order placement from admin panel with Simple product

### Manual testing scenarios (*)

1. none.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by ishakhsuvarov — CHANGES_REQUESTED** [review]
Since this test only relates to the In Store Pickup feature It would be appropriate to place it to the appropriate module in `InventoryInStorePickup` family and change target branch to `store-pickup` where all development is happening at the moment.

**Comment by nmalevanec** [comment]
@ishakhsuvarov , done.

File: app/code/Magento/InventoryInStorePickup/Test/Mftf/Test/InStorePickupOrderPlacementWithSimpleProductAdditionalStockTest.xml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ishakhsuvarov) [inline app/code/Magento/InventoryInStorePickup/Test/Mftf/Test/InStorePickupOrderPlacementWithSimpleProductAdditionalStockTest.xml]: I'd suggest putting mftf test related content into `*AdminUi` directories.

**Review by ishakhsuvarov — CHANGES_REQUESTED** [review]

File: app/code/Magento/InventoryInStorePickup/Test/Mftf/Test/InStorePickupOrderPlacementWithSimpleProductAdditionalStockTest.xml (hunk 1 above)
Comment (nmalevanec) [reply app/code/Magento/InventoryInStorePickup/Test/Mftf/Test/InStorePickupOrderPlacementWithSimpleProductAdditionalStockTest.xml]: Done. Test moved to InventoryInStorePickupAdminUi

**Comment by ishakhsuvarov** [comment]
Closing, merged into https://github.com/magento/inventory/pull/2390

**Pull request closed without merging**